daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/Convert.hs

```diff
@@ -757,13 +757,14 @@ convertExpr env0 e = do
         -- work. Constructor workers are not handled (yet).
         | Just m <- nameModule_maybe $ varName x
         , Just con <- isDataConId_maybe x
-        , not ("$W" `isPrefixOf` is x)
+        -- , not ("$W" `isPrefixOf` is x)
```

Comment:
Not a fan of arbitrarily commented out code

Reply:
That was an accident. Removed.